bolt/test/X86/fdata-escape-chars.c

```diff
@@ -0,0 +1,69 @@
+volatile int var;
+
+void symb_w_whitespace() { var = 1; }
+
+void symb_backslash_b() {
+  symb_w_whitespace();
+  var = 2;
+}
+
+static void static_symb_backslash_b() {
+  symb_w_whitespace();
+  var = 3;
+}
+
+int main() {
+  static_symb_backslash_b();
+  symb_w_whitespace();
+  for (int i = 0; i < 2; ++i)
+    symb_backslash_b();
+}
+
```

Comment:
My compiler is putting static_symb_backslash_b after main in the output object, and it causing this test to fail because it expects BOLT to print these symbols in order. In order to make this test stable, I think we need to rewrite them in assembly and do not use host_cc to build it.  (Feel free to use it to link, though).

Reply:
Yes, sorry, I forgot about this possibility :) I've decided to rewrite it using LLVM IR (actually just used clang -emit-llvm). AFAIU we can use host_cc to assemble and link the input, it should not optimize and reorder it at this point. I've check few compiler versions it it looks good, could you please check it? Thank you :)

Reply:
No problem! Sounds good!